includes/wikia/models/NavigationModel.class.php

```diff
@@ -82,6 +82,15 @@ public function clearMemc( $key = self::WIKIA_GLOBAL_VARIABLE ) {
 		WikiaDataAccess::cachePurge(
 			$this->getMemcKey( $key )
 		);
+
+		// Purging mcache set by invocation of NavigationModel::getTree() by NavigationModel::getLocalNavigationTree();
+		WikiaDataAccess::cachePurge(
+			$this->getTreeMemcKey( self::TYPE_MESSAGE, self::WIKI_LOCAL_MESSAGE, implode( [
+				self::LOCALNAV_LEVEL_1_ITEMS_COUNT,
+				self::LOCALNAV_LEVEL_2_ITEMS_COUNT,
+				self::LOCALNAV_LEVEL_3_ITEMS_COUNT
+			], '-' ) )
```

Comment:
Also, `getTreeMemcKey` already implodes the method arguments. Just pass in all the ingredients.